openssl/src/ssl/mod.rs

```diff
@@ -1930,6 +1948,24 @@ impl ToOwned for SslSessionRef {
     }
 }
 
+#[allow(non_camel_case_types)]
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum SrtpProfileId {
+    SRTP_AES128_CM_SHA1_80,
+    SRTP_AES128_CM_SHA1_32,
+    SRTP_AES128_F8_SHA1_80,
+    SRTP_AES128_F8_SHA1_32,
+    SRTP_NULL_SHA1_80,
+    SRTP_NULL_SHA1_32,
+    Other(usize),
+}
+
+#[derive(Debug, PartialEq)]
+pub struct SrtpProfile<'a> {
```

Comment:
Rather than defining a Rust struct for this, I think we should return an `&StrpProfileRef` and have methods on that to grab the ID and name.